Topcoder/FoldingPaper2.cc

```diff
@@ -0,0 +1,61 @@
+#include <iostream>
+#include <vector>
+#include <cmath>
+#include <utility>
+#define oo 1e8
```

Comment:
Use `const int oo = (int)1e8`. As it is `oo` is considered as a double, and that's not I think what you want.